docs/blog/design-philosophy/index.qmd

```diff
@@ -0,0 +1,245 @@
+
+![](./uruk_tablet_with_annotations.gif)
+
+<p style="font-size: 12px;">Drawing of clay tablet from Sumerian city of Uruk, circa 3200-3000 BCE. Uruk III Tablet (MSVO 3, 51, Louvre Museum, Paris, France). Annotated with the meanings of the columns, rows, and cells.</p>
+
+The inscribed grid in the ca. 3200-3000 BCE tablet creates boxes and each one of them represented an accounting unit. Inside each box was an ideogram (a symbol that represented a word or idea) and a numerical value representing a quantity. Now here is where it gets super interesting: each row corresponds to an individual, two columns contain entries for malt and barley; subtotals are given in the third column; and the left-most box displays the grand total. This has all the elements of a summary table! In addition, no formal language was used to explain the relationships within the tablet/table. Instead, the grid structure itself provided all that. As a bonus, the table has a footer as the bottom row contains the name of the official in charge. This is a visionary table we've got here.
+
+Zooming ahead about a thousand years, you start to see more systematically structured tables. Here's a photo of a cuneiform tablet that was originally from Mesopotamia (from the Temple of Enlil at Nippur, ca. 1850 BCE)[^6].
+
+![](./nippur_cuneiform_tablet.png)
+
+<p style="font-size: 12px;">Cuneiform tablet, temple of Enlil at Nippur, (CBS 3323, University of Pennsylvania).</p>
+
+You can see right away that there is a more regular grid and, if you probe deeper, there are more similarities than differerences with the tables of today. This contains sources of revenue and monthly disbursements for nearly 50 temple personnel. While it can be difficult to see it, you should take it on good authority[^7] that the following table elements are present:
+
+- column headings (month names) and row titles (names/professions of individuals).
+- cells with no information (look at the blank or smooth cells along rows)
+- numerical values in the cells
+- subtotals for each individual are every six months
+- grand totals adjacent to row labels
+- annotations with explanatory notes under those columns containing totals
+
+Later on, tables were less inscribed on clay and more on wax tablets, papyrus, paper, and eventually tables became computerized. What a trip it has been for tables!
+
+### Midcentury modern tables
+
+There's a prehistory to modern, computerized tables and perhaps the best period for tables was around the middle of the 20th century. Technologies for table (and surrounding document) preparation included the offset printer, the typewriter, and varitype[^8] (my favorite). While constrained by the limited space available on pages, tabular design at this point had many solutions for either fitting tables into single, variously-sized pages or distributing the table content across multiple pages.
```

Comment:
The piece I latched onto was that this time was the best period for tables. But I don't know why. I don't know why this paragraph said that and then listed technology, etc.. (is this the best period for tables because of how they were printed, or something else?).

I would address what made this the best period for tables directly in this paragraph, and then push everything to a new paragraph. (If the answer is that these formats induced something nice in tables, I could see them being a part of it!).